src/main/java/com/celements/web/plugin/cmd/DocFormCommand.java

```diff
@@ -319,24 +321,28 @@ String getFieldFromProperty(PropertyClass propclass, String field) {
 
   private BaseClass getBaseClass(String className, XWikiContext context) {
```

Comment:
drop not used XWikiContext parameter (signature change only because it is a private method)